api/src/services/sso-login.js

```diff
@@ -74,23 +57,27 @@ const oidcServerSConfig = async () => {
 
   const execute = allow_insecure_requests ? [client.allowInsecureRequests] : [];
   const discoveryUrl = new URL(discovery_url);
-  const idServerConfig = await authServerCallRetry(
-    () => client.discovery(discoveryUrl, client_id, clientSecret, null, { execute })
-  );
+  const discovery = () => client.discovery(discoveryUrl, client_id, clientSecret, null, { execute });
+  const idServerConfig = await authServerCallRetry(discovery);
 
   const {
     issuer,
     authorization_endpoint,
     token_endpoint,
+    grant_types_supported,
+    response_types_supported,
+    claims_supported,
+    scopes_supported,
   } = idServerConfig.serverMetadata();
-
-  const connectionMetadata = `
-  issuer: ${issuer}
-  authorization_endpoint: ${authorization_endpoint}
-  token_endpoint: ${token_endpoint}
-  `;
-
-  logger.debug(`Authorization server config auth config loaded successfully. ${connectionMetadata}`);
+  logger.debug(`Authorization server config loaded: ${JSON.stringify({
+    issuer,
+    authorization_endpoint,
+    token_endpoint,
+    grant_types_supported,
+    response_types_supported,
+    claims_supported,
+    scopes_supported,
+  })}`);
```

Comment:
Added a few more things to this. Really I just looked at the data coming back from `serverMetadata` and tried to pull out anything that seems relevant to our flow.  All of this could be very useful when debugging OIDC connection issues.